lib/Data/Database.php

```diff
@@ -562,10 +576,28 @@ private static function _select($sql, array $params, $firstOnly = false)
     {
         $statement = self::$_db->prepare($sql);
         $statement->execute($params);
-        $result = $firstOnly ?
-            $statement->fetch(PDO::FETCH_ASSOC) :
-            $statement->fetchAll(PDO::FETCH_ASSOC);
+        if ($firstOnly) {
+            $result = $statement->fetch(PDO::FETCH_ASSOC);
+        } elseif (self::$_type === 'oci') {
+            // workaround for https://bugs.php.net/bug.php?id=46728
+            $result = array();
+            while ($row = $statement->fetch(PDO::FETCH_ASSOC)) {
+                $result[] = $row;
```

Comment:
```suggestion
                $result[] = array_map('self::_sanitizeClob', $row);
```

This fixes the comment display issue

Reply:
Right, same as for the exec